india_compliance/gst_india/utils/e_invoice.py

```diff
@@ -685,7 +685,7 @@ def update_item_details(self, item_details, item):
         )
 
         if self.doc.is_reverse_charge:
-            item_details["total_value"] = item.taxable_value
+            item_details["total_value"] = abs(self.rounded(item.taxable_value, 3))
```

Comment:
```suggestion
            item_details["total_value"] = abs(self.rounded(item.taxable_value, 2))
```
precision should be 2 as the regex is `^\d+\.?\d{0,2}$`

Reply:
I tried to keep it consistent with above code.

"unit_rate": (
                    abs(self.rounded(item.taxable_value / item.qty, 3))

Reply:
> I tried to keep it consistent with above code.
> 
> "unit_rate": ( abs(self.rounded(item.taxable_value / item.qty, 3))

but wouldn't it throw the error from GST API? as it only accepts precision 2